### PR TITLE
an/SKY-11: Profile-edit page design 

### PR DESCRIPTION
**Pull request by anurmamedov**
1. Fontawesome icons added and profile page design created
large size profile-edit page
medium size profile-edit page
small size profile-edit page

### CODE REVIEW

File: templates/profile-edit.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
+    <body>
```
Comment (Aymammet) [inline templates/profile-edit.html:1]: can be removed

File: templates/profile-edit.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                                </div>
+                                <!-- Save changes button-->
+                                <button class="btn btn-primary" type="button">Save changes</button>
+                            </form>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </body>
```
Comment (Aymammet) [inline templates/profile-edit.html:77]: can be removed

**Review by Aymammet — CHANGES_REQUESTED** [review]

File: templates/profile-edit.html (hunk 2 above)
Comment (anurmamedov) [reply templates/profile-edit.html]: removed

**Review by anurmamedov — COMMENTED** [review]
Profile-edit page body removed

**Review by Aymammet — CHANGES_REQUESTED** [review]
Your task is about profile edit page design but there is profile create.html file. 
Shouldn't they be seperated?

**Comment by anurmamedov** [comment]
it is post  edit. i called commit by mistake profile edit.  It is post edit

**Review by Aymammet — APPROVED** [review]

**Pull request merged**